scripts/src/main/resources/scripts/command/python

```diff
@@ -0,0 +1,196 @@
+      make -j 8
+      make altinstall
+      make clean
+      make distclean
+      
+    elif doIsWindows
+    then
+      doEcho "Getting pip install file"
+      # https://github.com/pypa/get-pip
+      curl -sSL https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+      doInstall "-" "${PYTHON_HOME}" "python" "${PYTHON_VERSION}" "" "python"
+      doRunPython "get-pip.py"
+      rm get-pip.py
+      if [ -a "$(find "${PYTHON_HOME}" -name '*._pth')" ]
+      then
+        PTH_FILE="$(find "${PYTHON_HOME}" -name '*._pth')"
+        doEcho "Freeing Python from embedded Mode..."
+        sed -i -e 's/#import site/import site/g' "${PTH_FILE}"
+      else
+        doEcho "Error. Couldn't find _pth file."
+      fi
+      
+      # rename folder from Scripts to bin
+      if [ -d "${PYTHON_HOME}/Scripts" ]
+      then
+        mv "${PYTHON_HOME}"/Scripts "${PYTHON_HOME}"/bin
+      fi
+      
+    else
+      doInstallWithPackageManager "apt-get install -y make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev" \
+                                  "yum -y groupinstall \"Development Tools\"" \
+                                  "zypp install automake fdupes gcc gcc-c++ gcc-fortran gdbm-devel gettext-tools gmp-devel intltool libbz2-devel libexpat-devel libffi-devel libnsl-devel lzma-devel make ncurses-devel netcfg openssl-devel pkgconfig readline-devel sqlite-devel xz zlib-devel"
+      
+      
+      doInstall "-" "${PYTHON_HOME}" "python" "${PYTHON_VERSION}" "" "Python"
+      "${PYTHON_HOME}"/configure --enable-optimizations --with-ensurepip=install --prefix="${PYTHON_HOME}"
+      make -j 8
+      make altinstall
+      make clean
+      make distclean
+ 
+    fi
+    
+    if [ -d "${DEVON_IDE_HOME}/Python" ]
+    then
+      rm -r "${DEVON_IDE_HOME}/Python"
+    fi
+    
+  fi
+
+  # set bin locations
+  if ! doIsWindows
+  then
+    PYTHON_EXE=$(find "${PYTHON_BIN}" -name "python3.*" | sort | head -1)
+    PIP_EXE=$(find "${PYTHON_BIN}" -name "pip*" | sort | head -1)
+  else
+    PYTHON_EXE="${PYTHON_HOME}/python.exe"
+    PIP_EXE="${PYTHON_BIN}/pip.exe"
+    PIP3_EXE="${PYTHON_BIN}/pip3.exe"
+  fi
+ 
+  # check version
+  if [ "${1}" != "silent" ] && ! doIsQuiet
+  then
+    doRunCommand "${PYTHON_EXE} --version"
+  fi 
+  
+
+}
+
+function setPath() {
+  local path_bind="PATH=\$PATH:"
+  local set_path="${path_bind}${PYTHON_BIN}"
+  if ! grep -q "${set_path}" "${DEVON_IDE_HOME}/conf/devon.properties"
+    then
+      export "${set_path}"
+      echo -e "\n${set_path}" >> "${DEVON_IDE_HOME}/conf/devon.properties"
+      doEcho "Python and pip path is added to ${DEVON_IDE_HOME}/conf/devon.properties"
+  fi
+}
+
+function doRunPython() {
+  doSetup silent
+  if doIsQuiet
+  then
```

Comment:
If I use `devon python pip install azure-cli` I get the following error: `Working directory azure-cli does not exist`

Without `doRunCommand` the installation of azure-cli works.